src/main/g8/build.sbt

```diff
@@ -3,23 +3,22 @@ lazy val root = (project in file(".")).
   settings(
     inThisBuild(List(
       organization := "$organization$",
-      scalaVersion := "2.11.8"
+      scalaVersion := "2.11.11"
```

Comment:
So I generally keep 2.11.8 since that's the version Spark is currently built with. What is the a reason for 2.11.11?